### DISCUSSION TITLE
View collection as relation source?

### DISCUSSION THREAD

**Discussion by alancwoo** [pocketbase/pocketbase#4731] [Q&A]
I have a series of game records such as:

`records`: `id`, `userId`, `username`, `score`

I created a view collection to find all the distinct `userId` values and was hoping to create another collection where I can manually relate users that are the same person but have a different account/userId.

Is there any way to specify the view collection as the source for the `relation` field?

**Comment by ganigeorgiev** [comment]
> Is there any way to specify the view collection as the source for the relation field?

No, this is not supported because the views are computed/dynamic and there is no guarantee that the resulting ids will not change between queries.
